src/main/java/io/github/railroad/editor/ModelEditor.java

```diff
@@ -0,0 +1,133 @@
+package io.github.railroad.editor;
+
+import javafx.animation.KeyFrame;
+import javafx.animation.Timeline;
+import javafx.application.Application;
+import javafx.scene.AmbientLight;
+import javafx.scene.Group;
+import javafx.scene.PerspectiveCamera;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ToggleButton;
+import javafx.scene.input.MouseButton;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.HBox;
+import javafx.scene.paint.Color;
+import javafx.scene.paint.PhongMaterial;
+import javafx.scene.shape.Box;
+import javafx.stage.Stage;
+import javafx.util.Duration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// TODO make this actually part of the IDE, not a stand alone run thingy
+// TODO make editor
+// TODO make exporter.
+// TODO a lot of things.
+
+//TODO: get rid of this
```

Comment:
You can get rid of this now, I was the original author but this is just a test that was done very badly :(